app/templates/components/forms/orders/order-form.hbs

```diff
@@ -278,13 +278,25 @@
         <i class="ticket icon"></i>
         {{t 'Acknowledgement'}}
       </h4>
+       {{#if this.isPaidOrder}}
+        <p>
+        {{t
+          'By clicking "Proceed to Checkout", I acknowledge that I have read and that I agree with the '}} 
+           <a href='/terms'>terms of services</a>{{t ' and ' }}<a href='/privacy'>privacy policy</a> {{ ' on this website.'}}      
+         </p>
+      <div class="center aligned">
+        <button type="submit" class="ui teal submit button">{{t 'Proceed to Checkout'}}</button>
+      </div>
+      {{else}}
       <p>
         {{t
-          'By clicking "Pay Now", I acknowledge that I have read and that I agree with all the terms of services and privacy policy of Open Event.'}}
+          'By clicking "Order Now", I acknowledge that I have read and that I agree with the '}} 
+           <a href='/terms'>terms of services</a>{{t ' and ' }}<a href='/privacy'>privacy policy</a> {{ ' on this website.'}}
```

Comment:
Repetition. The only difference is "Proceed to Checkout" and "Order Now"

Reply:
This is what is asked in the issue.
It's same there too, only difference is Order Now and Proceed to Checkout

Reply:
I know. There should be no duplication. This can be achieved without that

Reply:
![Screenshot from 2020-11-20 18-56-57](https://user-images.githubusercontent.com/55341933/99805285-49cea180-2b62-11eb-892e-87c633e5809b.png)


Would this work?

Reply:
Try